src/mote/modules/find.py

```diff
@@ -26,7 +26,9 @@
 import urllib.parse as ulpr
 from datetime import datetime
 
-directory_path = os.path.dirname("/srv/web/meetbot/")
+from flask import current_app as app
+
+directory_path = os.path.dirname("/srv/web/meetbot")
```

Comment:
The two lines have been moved into the function. Otherwise, the value is accessed before the application is initialized. This causes the flask to give out an error.